### ISSUE TITLE
Compact the db and views before starting an upgrade

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#4296]
This will free up some space which might save us from running out on upgrade. It can be run while API is still running so won't increase the downtime.

**Comment by SCdF** [comment]
This is already covered here: https://github.com/medic/medic-webapp/issues/3993#issuecomment-371742490
